ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/CommandTest.java

```diff
@@ -70,7 +71,8 @@ public void shouldSerializeDeserializeCorrectly() throws IOException {
     final Command command = new Command(
         "test statement;",
         Collections.singletonMap("foo", "bar"),
-        Collections.singletonMap("biz", "baz"));
+        Collections.singletonMap("biz", "baz"),
```

Comment:
Should probably add a test to ensure the new checksum is serialized and deserialized.